src/actions/indexing/indexable-prepare-indexation-action.php

```diff
@@ -3,61 +3,61 @@
 namespace Yoast\WP\SEO\Actions\Indexing;
 
 use Yoast\WP\SEO\Helpers\Date_Helper;
-use Yoast\WP\SEO\Helpers\Options_Helper;
+use Yoast\WP\SEO\Helpers\Indexing_Helper;
 use Yoast\WP\SEO\Integrations\Admin\Indexing_Notification_Integration;
 use Yoast_Notification_Center;
 
 /**
- * Action for preparing the indexable indexation routine.
+ * Action for preparing the indexable indexing routine.
  */
 class Indexable_Prepare_Indexation_Action {
 
 	/**
-	 * The options helper.
+	 * The notification center.
 	 *
-	 * @var Options_Helper
+	 * @var Yoast_Notification_Center
 	 */
-	protected $options;
+	protected $notification_center;
 
 	/**
 	 * The date helper.
 	 *
 	 * @var Date_Helper
 	 */
-	protected $date;
+	protected $date_helper;
 
 	/**
-	 * The notification center.
+	 * The indexing helper.
 	 *
-	 * @var Yoast_Notification_Center
+	 * @var Indexing_Helper
 	 */
-	protected $notification_center;
+	protected $indexing_helper;
 
 	/**
-	 * Action for preparing the indexable indexation routine.
+	 * Action for preparing the indexable indexing routine.
 	 *
-	 * @param Options_Helper            $options             The options helper.
-	 * @param Date_Helper               $date                The date helper.
+	 * @param Date_Helper               $date_helper         The date helper.
+	 * @param Indexing_Helper           $indexing_helper     The indexing helper.
 	 * @param Yoast_Notification_Center $notification_center The notification center.
 	 */
 	public function __construct(
-		Options_Helper $options,
-		Date_Helper $date,
+		Date_Helper $date_helper,
+		Indexing_Helper $indexing_helper,
 		Yoast_Notification_Center $notification_center
 	) {
-		$this->options             = $options;
-		$this->date                = $date;
+		$this->date_helper         = $date_helper;
+		$this->indexing_helper     = $indexing_helper;
 		$this->notification_center = $notification_center;
 	}
 
 	/**
-	 * Prepare the indexable indexation routine.
+	 * Prepare the indexable indexing routine.
 	 *
 	 * @return void
 	 */
 	public function prepare() {
-		$this->options->set( 'indexing_first_time', false );
-		$this->options->set( 'indexation_started', $this->date->current_time() );
+		$this->indexing_helper->set_first_time( false );
+		$this->indexing_helper->set_started( $this->date_helper->current_time() );
```

Comment:
You might think of a method in the indexing_helper: `start` that contains this logic. This also means we can remove the date_helper as dependency of this class. It's a minor thing.

Reply:
Done.